modules/contact-form/grunion-contact-form.php

```diff
@@ -2017,6 +2017,11 @@ static function parse( $attributes, $content ) {
 			$r .= $form->body;
 			$r .= "\t<p class='contact-submit'>\n";
 
+			$gutenberg_submit_button_classes = '';
+			if ( isset( $attributes['submitButtonClasses'] ) && $attributes['submitButtonClasses'] ) {
```

Comment:
In this case I believe you should be able to simplify and just check for `if ( ! empty( $attributes['submitButtonClasses'] )`

```suggestion
			if ( ! empty( $attributes['submitButtonClasses'] ) ) {
```